src/Sarif.Driver/Sdk/DefaultDriverOptions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif.Driver.Sdk
```

Comment:
Nit:  Namespace should probably just be Microsoft.CodeAnalysis.Sarif.Driver

Reply:
I see now that the code is in an inconsistent condition. I went ahead and removed all reference to the SDK namespace.

---
In reply to: [169829946](https://github.com/Microsoft/sarif-sdk/pull/791#discussion_r169829946) [](ancestors = 169829946)